.github/workflows/train.yml

```diff
@@ -0,0 +1,61 @@
+name: "Run training"
+
+# Trigger the workflow on push for the master branch
+on:
+  push:
```

Comment:
Think this makes sense for now (only train once a pull request is merged right?)  
Would we want the same action or different for `on: pull_request`, run a training and make sure it's ok so that a reviewer can see the outcome before allowing merge to master? No clue how we would gather the output from the training etc, just trying to understand the _potential_ actions we could have

Reply:
Sure, this is totally possible. We can run training even on every commit, it's just a question of consumed CPU resources. We may have one MLFlow instance for PR training and one for `master` trainings in order to not confuse them. Also, I guess every branch can be its own experiment in ML flow.